sources/packages/backend/libs/integrations/src/sfas-integration/sfas-files/sfas-individual-record.ts

```diff
@@ -95,78 +101,90 @@ export class SFASIndividualRecord extends SFASRecordIdentification {
    * The date when a PPD status is effective (individual.ppd_status_dte).
    */
   get ppdStatusDate(): Date | null {
-    return parseDate(this.line.substring(87, 95));
+    return parseDate(this.line.substring(95, 103));
   }
   /**
    * The most recent, active Master Student Loan Agreement Number (loan_agreement_request.msfaa_agreement_number).
    */
   get msfaaNumber(): number | null {
-    return +this.line.substring(95, 105) || null;
+    return +this.line.substring(103, 113) || null;
   }
   /**
    * The most recent, active Master Student Loan Agreement signed date (loan_agreement_request.loan_agreement_signed_dte).
    */
   get msfaaSignedDate(): Date | null {
-    return parseDate(this.line.substring(105, 113));
+    return parseDate(this.line.substring(113, 121));
+  }
+  /**
+   * Vendor/supplier number assigned to an individual by CAS	(individual.supplier_no).
+   */
+  get supplierNumber(): Date | null {
+    return parseDate(this.line.substring(121, 128));
+  }
+  /**
+   * Site number assigned to an individual by CAS (individual.site_no).
+   */
+  get siteNumber(): Date | null {
+    return parseDate(this.line.substring(128, 131));
```

Comment:
Same here. we should not read the fields that is not used by the system. 

https://github.com/bcgov/SIMS/pull/3633/files#r1718997491